docs/papers/utils/minipaper_cracking_symmetry.py

```
@@ -0,0 +1,353 @@
+    table_data = [
+        [name, f"{initial:.4f}", f"{opt:.4f}"]
+        for name, initial, opt in zip(param_names, initial_vals, optimized_vals)
+    ]
+    table_data.append(["Mean Squared Error", "", f"{mse:.6f}"])
+
+    print(tabulate(table_data, headers=["Parameter", "Initial", "Optimized"], tablefmt="github"))
+
+
+import numpy as np
+
+def generate_loss_surfaces(
```

Ruff (0.11.9)

Missing return type annotation for public function `generate_loss_surfaces`

(ANN201)